ies-gearmand/recipes/service.rb

```diff
@@ -20,7 +20,7 @@
   end
 end
 
-cookbook_file "/etc/init/#{name}.override" do
```

Comment:
Why?

Reply:
because of this:
```
==> web: [2017-10-23T14:19:35+00:00] INFO: Processing service[gearman-job-server] action restart (ies-gearmand::service line 11)
==> web: [2017-10-23T14:19:35+00:00] DEBUG: service[gearman-job-server] did not find /etc/init/gearman-job-server.conf
==> web: [2017-10-23T14:19:35+00:00] DEBUG: Executing /sbin/start gearman-job-server
==> web: [2017-10-23T14:19:35+00:00] DEBUG: ---- Begin output of /sbin/start gearman-job-server ----
==> web: [2017-10-23T14:19:35+00:00] DEBUG: STDOUT:
==> web: [2017-10-23T14:19:35+00:00] DEBUG: STDERR: start: Unknown job: gearman-job-server
==> web: [2017-10-23T14:19:35+00:00] DEBUG: ---- End output of /sbin/start gearman-job-server ----
==> web: [2017-10-23T14:19:35+00:00] DEBUG: Ran /sbin/start gearman-job-server returned 1
==> web:
==> web: ================================================================================
==> web: Error executing action `restart` on resource 'service[gearman-job-server]'
==> web: ================================================================================
==> web:
==> web:
==> web: Chef::Exceptions::Exec
==> web: ----------------------
==> web: /sbin/start gearman-job-server returned 1, expected 0
==> web: ---- Begin output of /sbin/start gearman-job-server ----
==> web: STDOUT: STDERR: start: Unknown job: gearman-job-server
==> web: ---- End output of /sbin/start gearman-job-server ----
==> web:
==> web:
==> web:
==> web: Resource Declaration:
==> web: ---------------------
==> web: # In /tmp/vagrant-chef/f5705899f051c84b079b835ca4ce852e/cookbooks/ies-gearmand/recipes/service.rb
==> web:
==> web:  11: service name do
==> web:  12:   action :nothing
==> web:  13:   provider service_provider
==> web:  14: end
==> web:  15:
==> web:
==> web:
==> web:
==> web: Compiled Resource:
==> web: ------------------
==> web: # Declared in /tmp/vagrant-chef/f5705899f051c84b079b835ca4ce852e/cookbooks/ies-gearmand/recipes/service.rb:11:in `from_file'
==> web:
==> web: service("gearman-job-server") do
==> web:   provider Chef::Provider::Service::Upstart
==> web:   action [:nothing]
==> web:   supports {:restart=>false, :reload=>false, :status=>false}
==> web:   retries 0
==> web:   retry_delay 2
==> web:   service_name "gearman-job-server"
==> web:   pattern "gearman-job-server"
==> web:   cookbook_name :"ies-gearmand"
==> web:   recipe_name "service"
==> web: end
==> web:
==> web:
```

Reply:
@gilleyj @till We're still seeing this exact error, why did you revert this fix @gilleyj ?